src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -875,13 +875,18 @@ private uint getClassAttribs(CORINFO_CLASS_STRUCT_* cls)
         private uint getClassAttribsInternal(TypeDesc type)
         {
             // TODO: This method needs to implement:
-            //       1. GenericParameterType: CORINFO_FLG_GENERIC_TYPE_VARIABLE
-            //       2. Shared instantiation: IsCanonicalSubtype, IsRuntimeDeterminedSubtype: CORINFO_FLG_SHAREDINST
-            //       3. HasVariance: CORINFO_FLG_VARIANCE
-            //       4. Finalizer support: CORINFO_FLG_HAS_FINALIZER
+            //       1. Shared instantiation: IsCanonicalSubtype, IsRuntimeDeterminedSubtype: CORINFO_FLG_SHAREDINST
 
             CorInfoFlag result = (CorInfoFlag)0;
 
+            if (type.IsSignatureVariable)
```

Comment:
> This may actually rather be  `IsGenericParameter` 

The methods on `TypeHandle` in CoreCLR could use better comments :). `// VAR, MVAR` maps to a signature variable.

> ... and it does not seem to be even used in RyuJIT

I saw a reference to it in importer.cpp, but didn't investigate further.

I'll remove this part, including the TODO. If we want to support verification, it will be separate feature work.